src/test/java/org/sagebionetworks/bridge/models/schedules/IntervalActivitySchedulerTest.java

```diff
@@ -52,6 +52,36 @@ public void after() {
         DateTimeUtils.setCurrentMillisSystem();
     }
     
+    @Test
+    public void recurringScheduleBugInIndianTimeZone() {
+        Schedule schedule = new Schedule();
+        schedule.setScheduleType(RECURRING);
+        schedule.setInterval("P1D");
+        schedule.setExpires("P1D");
+        schedule.addTimes("08:00");
+        schedule.addActivity(TestUtils.getActivity3());
+
+        DateTime now = DateTime.parse("2019-12-18T23:52:33.462+05:30");
+        DateTimeUtils.setCurrentMillisFixed(now.getMillis());
+
+        events.put("enrollment", DateTime.parse("2019-11-28T22:31:41.270-07:00"));
+
+        ScheduleContext context = new ScheduleContext.Builder()
+            .withStudyIdentifier(TEST_STUDY)
+            .withInitialTimeZone(DateTimeZone.forOffsetHours(-7))
+            .withStartsOn(DateTime.parse("2019-12-18T00:00:00.000+05:30"))
+            .withEndsOn(DateTime.parse("2019-12-19T00:00:00.000+05:30"))
+            .withEvents(events).build();
+        scheduledActivities = schedule.getScheduler().getScheduledActivities(plan, context);
+
+        assertEquals(scheduledActivities.size(), 2);
+        assertEquals(scheduledActivities.get(0).getScheduledOn().toString(), "2019-12-17T08:00:00.000+05:30");
+        assertEquals(scheduledActivities.get(0).getExpiresOn().toString(), "2019-12-18T08:00:00.000+05:30");
+        assertEquals(scheduledActivities.get(1).getScheduledOn().toString(), "2019-12-18T08:00:00.000+05:30");
+        assertEquals(scheduledActivities.get(1).getExpiresOn().toString(), "2019-12-19T08:00:00.000+05:30");
+        DateTimeUtils.setCurrentMillisSystem();
```

Comment:
Minor: This call to setCurrentMillisSystem() is redundant with after().

Reply:
Removed